test/functional/wallet_send.py

```diff
@@ -0,0 +1,339 @@
+#!/usr/bin/env python3
+# Copyright (c) 2020 The Bitcoin Core developers
+# Distributed under the MIT software license, see the accompanying
+# file COPYING or http://www.opensource.org/licenses/mit-license.php.
+"""Test the send RPC command."""
+
+from decimal import Decimal, getcontext
+from test_framework.authproxy import JSONRPCException
+from test_framework.test_framework import BitcoinTestFramework
+from test_framework.util import (
+    assert_equal,
+    assert_fee_amount,
+    assert_greater_than,
+    assert_raises_rpc_error
+)
+
+class WalletSendTest(BitcoinTestFramework):
+    def set_test_params(self):
+        self.num_nodes = 2
+        # whitelist all peers to speed up tx relay / mempool sync
+        self.extra_args = [
+            ["-whitelist=127.0.0.1","-walletrbf=1"],
+            ["-whitelist=127.0.0.1","-walletrbf=1"],
+        ]
+        getcontext().prec = 8 # Satoshi precision for Decimal
+
+    def skip_test_if_missing_module(self):
+        self.skip_if_no_wallet()
+
+    def test_send(self, from_wallet, to_wallet=None, amount=None, data=None,
+                  arg_conf_target=None, arg_estimate_mode=None,
+                  conf_target=None, estimate_mode=None, add_to_wallet=None,psbt=None,
+                  inputs=None,add_inputs=None,change_address=None,change_position=None,change_type=None,
```

Comment:
nit: spaces after each arg, here and below

Reply:
`,[a-zA-Z0-9]+` found a bunch more